pkg/virt-api/webhooks/mutating-webhook/mutators/vmi-mutator_test.go

```diff
@@ -414,42 +414,46 @@ var _ = Describe("VirtualMachineInstance Mutator", func() {
 		),
 	)
 
-	DescribeTable("should add the default network interface",
-		func(iface string) {
-			expectedIface := "bridge"
-			switch iface {
-			case "masquerade":
-				expectedIface = "masquerade"
-			case "slirp":
-				expectedIface = "slirp"
-			}
+	Context("should add network interfaces", func() {
```

Comment:
Perhaps use this: `When("Kubevirt has network-interface defined", ...`

Reply:
Done